src/equations/acoustic_perturbation_2d.jl

```diff
@@ -344,8 +350,11 @@ end
                                                               equations::AcousticPerturbationEquations2D)
     λ = dissipation.max_abs_speed(u_ll, u_rr, orientation_or_normal_direction,
                                   equations)
-    diss = -0.5 * λ * (u_rr - u_ll)
-    z = zero(eltype(u_ll))
+
+    RealT = eltype(u_ll)
+    diss = -convert(RealT, 0.5) * λ * (u_rr - u_ll)
+    z = zero(RealT)
```

Comment:
```suggestion
    RealT = eltype(u_ll)
    diss = -0.5f0 * λ * (u_rr - u_ll)
    z = zero(RealT)
```
As discussed earlier, `Float32` values like `0.5f0` will be promoted automatically so that no explicit conversion is required.

Reply:
This is for consistency and avoiding using explicit `0.5f0`, do you still prefer `0.5f0`?

Reply:
Yes. We will use it quite often and it's easier to read, I think.